tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -73,6 +73,10 @@ def delete_data_set(hosts, data_set_name):
 def delete_file(hosts, path):
     hosts.all.file(path=path, state="absent")
 
+def delete_remnants(hosts):
+    hosts.all.shell(cmd="drm 'ANSIBLE.*'")
+    hosts.all.shell(cmd="drm 'TEST.*'")
```

Comment:
Question about this TEST.*, which data sets get deleted by it ? 
Also, why is this needed if tests do delete the data sets in their finally section? Is the module leaving temporary data sets behind?

Reply:
For the test `test_backup_and_restore_of_data_set` is fixed by adding a new hlq to ensure works properly and do not mesh with the dataset is not removed as the previous behavior, and yes as my changes for random dataset name make it almost impossible to found all datasets to be remove with a specific name

Reply:
makes sense